### PR TITLE
Feature gameboard

### PR DESCRIPTION
**Pull request by AbdurakhimovAziz**

### CODE REVIEW

File: src/app/components/cell/cell.component.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  selector: 'app-cell',
+  templateUrl: './cell.component.html',
+  styleUrls: ['./cell.component.scss'],
+})
+export class CellComponent{
+  @Input() cell!: Cell;
+  @Output() dragStarted: EventEmitter<void> = new EventEmitter();
+  @Output() dragEnded: EventEmitter<CdkDragEnd> = new EventEmitter();
+
+  constructor() {}
```
Comment (Morphine-medicine) [inline src/app/components/cell/cell.component.ts]: Constructor can be removed if unused

File: src/app/components/main/main.component.ts [hunk 2]
```diff
@@ -1,44 +1,12 @@
...
-  public sendMsg(): void {
-    this.websocket.send('message', 'some data');
-  }
-
-  public onSubmit(): void {
-    if (this.wsForm.valid && this.wsForm.value.event) {
-      this.websocket.send(this.wsForm.value.event, this.wsForm.value.data);
-    }
-  }
+  ngOnInit(): void {}
```
Comment (Morphine-medicine) [inline src/app/components/main/main.component.ts]: OnInit and constructors can be removed

File: src/app/components/game/game.component.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    this.addSubscription(
+      this.moveService.lastMove$.subscribe((move) => {
+        this.gameService.swapCurrentPlayer();
+      })
+    );
+
+    this.addSubscription(
+      this.moveService.lastMove$.subscribe((move: Move) => {
+        const figure = move.getMoveedFigure();
+        if (
```
Comment (Morphine-medicine) [inline src/app/components/game/game.component.ts:62]: Why do we need this if?

**Review by Morphine-medicine — CHANGES_REQUESTED** [review]
Looks really great!

File: src/app/components/game/game.component.ts (hunk 3 above)
Comment (AbdurakhimovAziz) [reply src/app/components/game/game.component.ts:62]: to set 'moved' property of these figures to true. There are certain mechanics, which are allowed only if those figures haven't moved yet

Comment (Morphine-medicine) [reply src/app/components/game/game.component.ts:62]: Oh, got it, thanks, can be merged now

**Pull request merged**